src/screens/Voucher/VoucherLoginScreen.tsx

```diff
@@ -57,6 +61,9 @@ export const VoucherLoginScreen = ({ navigation, route }: StackScreenProps<any>)
         // save auth token and member id to global state
         storeVoucherAuthToken(responseData.member.authentication_token);
         storeVoucherMemberId(responseData.member.id);
+        storeVoucherMemberLoginInfo(
+          JSON.stringify({ key: loginData.key, secret: loginData.secret })
```

Comment:
is `loginData` more than these two values? why not `JSON.stringify(loginData)`?

Reply:
Yes, you're right. `loginData` is enough here.

Reply:
https://github.com/smart-village-solutions/smart-village-app-app/pull/1102/commits/2e7a1781b0da877b496c88a97ac390aa9015c0c5